src/Driver/PCOV.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/*
+ * This file is part of the php-code-coverage package.
+ *
+ * (c) Sebastian Bergmann <sebastian@phpunit.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+namespace SebastianBergmann\CodeCoverage\Driver;
+
+use SebastianBergmann\CodeCoverage\RuntimeException;
+use SebastianBergmann\CodeCoverage\Filter;
+
+/**
+ * Driver for PCOV code coverage functionality.
+ *
+ * @codeCoverageIgnore
+ */
+final class PCOV implements Driver
+{
+    /**
+     * @throws RuntimeException
+     */
+    public function __construct(Filter $filter = null)
+    {
+	
+    }
+
+    /**
+     * Start collection of code coverage information.
+     */
+    public function start(bool $determineUnusedAndDead = true): void
+    {
+        \pcov\start();
+    }
+
+    /**
+     * Stop collection of code coverage information.
+     */
+    public function stop(): array
+    {
+	\pcov\stop();
+
+	$waiting = \pcov\waiting();
+	$collect  = [];
+
+	if ($waiting) {
+		$collect = \pcov\collect(\pcov\inclusive, $waiting);
+
+		if ($collect) {
+			\pcov\clear();
+		}
```

Comment:
Looks like the indentation is a bit off here, doesn't it?

Reply:
I can't see it ?

Reply:
```diff
diff --git a/src/CodeCoverage.php b/src/CodeCoverage.php
index 69b4bc0..62e28b4 100644
--- a/src/CodeCoverage.php
+++ b/src/CodeCoverage.php
@@ -13,8 +13,8 @@
 use PHPUnit\Runner\PhptTestCase;
 use PHPUnit\Util\Test;
 use SebastianBergmann\CodeCoverage\Driver\Driver;
-use SebastianBergmann\CodeCoverage\Driver\PHPDBG;
 use SebastianBergmann\CodeCoverage\Driver\PCOV;
+use SebastianBergmann\CodeCoverage\Driver\PHPDBG;
 use SebastianBergmann\CodeCoverage\Driver\Xdebug;
 use SebastianBergmann\CodeCoverage\Node\Builder;
 use SebastianBergmann\CodeCoverage\Node\Directory;
@@ -906,9 +906,9 @@ private function selectDriver(Filter $filter): Driver
             return new Xdebug($filter);
         }

-       if ($runtime->hasPCOV()) {
+        if ($runtime->hasPCOV()) {
             return new PCOV($filter);
-       }
+        }

         throw new RuntimeException('No code coverage driver available');
     }
diff --git a/src/Driver/PCOV.php b/src/Driver/PCOV.php
index 941650e..12f7643 100644
--- a/src/Driver/PCOV.php
+++ b/src/Driver/PCOV.php
@@ -9,7 +9,6 @@
  */
 namespace SebastianBergmann\CodeCoverage\Driver;

-use SebastianBergmann\CodeCoverage\RuntimeException;
 use SebastianBergmann\CodeCoverage\Filter;

 /**
@@ -21,7 +20,6 @@ final class PCOV implements Driver
 {
     public function __construct(Filter $filter = null)
     {
-
     }

     /**
@@ -37,19 +35,19 @@ public function start(bool $determineUnusedAndDead = true): void
      */
     public function stop(): array
     {
-       \pcov\stop();
+        \pcov\stop();

-       $waiting = \pcov\waiting();
-       $collect  = [];
+        $waiting  = \pcov\waiting();
+        $collect  = [];

-       if ($waiting) {
-               $collect = \pcov\collect(\pcov\inclusive, $waiting);
+        if ($waiting) {
+            $collect = \pcov\collect(\pcov\inclusive, $waiting);

-               if ($collect) {
-                       \pcov\clear();
-               }
-       }
+            if ($collect) {
+                \pcov\clear();
+            }
+        }

-       return $collect;
+        return $collect;
     }
 }
```

Reply:
Ha, sorry - I ran

```
$ composer global require friendsofphp/php-cs-fixer:^2.14.0
```

followed by

```
$ php-cs-fixer fix --diff --verbose
```

There were some unrelated fixes applied to a test case, which I have removed from the patch above.

Reply:
Indentation is off for me too. I think it's caused by the mix of tabs and spaces. Same in `CodeCoverage.php`.

Reply:
Already fixed that locally in a branch where I am preparing this before merging.

Reply:
Thanks people, but I'm going to just let Sebastian fix this upon merge, since he has to make minor edits anyway, and I don't want to make any more noise.

Reply:
Exactly. Would make things easier for me if this PR would not be altered anymore.

Reply:
ACK